spec/components/ReviewTile.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import 'jest-enzyme';
+import ReviewTile from '../../app/frontend/react/components/ReviewTile.js';
+
+describe('<ReviewTile/>', () => {
+  it('should render without throwing an error', () => {
+    const wrapper = mount(
+      <ReviewTile
+        userEmail="test@gmail.com"
+        rating="5"
+        body="so good"
+      />
+    );
+    expect(wrapper.find('.review-tile')).toContainClass('review-tile');
+  });
```

Comment:
Because this component renders some text, I'd like to see a couple more tests on on that.